vs_utils/scripts/public_data/get_class_data.py

```diff
@@ -0,0 +1,92 @@
+"""Build data frames for non-PCBA classification datasets."""
```

Comment:
What about something like "for classification datasets with separate files for active and inactive molecules (e.g. DUD-E)" instead? Other non-PCBA datasets could have different format.s